content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,135 @@
-<https://dashboard.stripe.com/account/apikeys>`_, or log into your Stripe dashboard and go to
-:menuselection:`Developers --> API Keys --> Standard Keys`.
+#. Go to `the API keys page on Stripe <https://dashboard.stripe.com/account/apikeys>`_, or log into
+   your Stripe dashboard and go to :menuselection:`Developers --> API Keys`.
+#. In the :guilabel:`Standard keys` section, copy the :guilabel:`Publishable key` and the
+   :guilabel:`Secret key` and save them for later.
+#. In Odoo, :ref:`navigate to the payment provider Stripe <payment_providers/supported_providers>`.
+#. In the :guilabel:`Credentials` tab, fill in the :guilabel:`Publishable Key` and
+   :guilabel:`Secret Key` fields with the values you previously saved.
 
 .. _stripe/webhook:
 
 Generate a webhook
 ------------------
 
-In case your **Webhook Signing Secret** is required to connect with your Stripe account, you can
-create a webhook either automatically or manually.
+If your **Webhook Signing Secret** is required to connect with your Stripe account, you can
+create a webhook automatically or manually.
 
 .. tabs::
    .. tab:: Create the webhook automatically
 
       Make sure your :ref:`Publishable and Secret keys <stripe/api_keys>` are filled in, then click
-      on the *Generate your Webhook* button.
+      :guilabel:`Generate your Webhook`.
 
    .. tab:: Create the webhook manually
 
-      Visit the `webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into your
-      Stripe dashboard and go to :menuselection:`Developers --> Webhooks`. Then, click on **Add
-      endpoint** in your **Hosted endpoints** and insert the following data into the pop-up form:
-
-      - | In the **Endpoint URL**, enter your Odoo database's URL followed by
-        | `/payment/stripe/webhook`.
-        | For example: `https://yourcompany.odoo.com/payment/stripe/webhook`
-      - At the end of the form, you can **Select events** to listen to. Click on it and, in the
-        **Checkout** section, select **checkout.session.completed**.
-
-        .. note::
-           It is possible to select other events, but they are currently not processed by Odoo.
-
-      When you click on **Add endpoint**, your Webhook is configured. You can then click on
-      **reveal** to display your signing secret.
+      #. Go to the `Webhooks page on Stripe <https://dashboard.stripe.com/webhooks>`_, or log into
+         your Stripe dashboard and go to :menuselection:`Developers --> Webhooks`.
+      #. In the :guilabel:`Hosted endpoints` section, click :guilabel:`Add endpoint`. Then, in the
+         :guilabel:`Endpoint URL` field, enter your Odoo database's URL, followed by
+         `/payment/stripe/webhook`, e.g., `https://yourcompany.odoo.com/payment/stripe/webhook`.
+      #.  Click :guilabel:`Select events` at the bottom of the form, then, in the
+          :guilabel:`Checkout` section, select :guilabel:`checkout.session.completed`, then click
```

Comment:
Since Odoo 16, there are actually more events than `checkout.session.completed` to select. The full list is hardcoded [here](https://github.com/odoo/odoo/blob/16.0/addons/payment_stripe/const.py#L38-L45) under the `HANDLED_WEBHOOK_EVENTS` constant. When forward-porting this PR, you'll need to monitor `HANDLED_WEBHOOK_EVENTS` over versions (both SaaS and major releases; use the dropdown on the left) and accordingly update the list of events in the documentation.